libs/sdk-ui-kit/src/utils/featureFlags.ts

```diff
@@ -0,0 +1,44 @@
+// (C) 2021 GoodData Corporation
+import { IFeatureFlags } from "@gooddata/goodstrap/lib/@next/Header/types";
```

Comment:
such type should already exist in UI SDK

Reply:
Yes, it is called `IUserWorkspaceSettings` or if that is too strict, there is also `ISettings` that is a base type for it.

Reply:
Thanks Dan and Ivan, I used ISettings instead of importing from goodstrap